### PR TITLE
Monday

### PR DESCRIPTION
**Pull request by rahulagarawal7**

### CODE REVIEW

File: src/screens/doctorDetailsScreen.jsx
Comment (aayushthoughtwin) [inline src/screens/doctorDetailsScreen.jsx:1]: use camel casing in styles and everywhere

File: src/screens/popularDoctorScreens/popularDoctor.jsx
Comment (aayushthoughtwin) [inline src/screens/popularDoctorScreens/popularDoctor.jsx:1]: why this much margin, explain?

File: src/components/backbutton.jsx
Comment (aayushthoughtwin) [inline src/components/backbutton.jsx:1]: again left and top without absolute

File: src/screens/popularDoctorScreens/popularDoctor.jsx
Comment (rahulagarawal7) [reply src/screens/popularDoctorScreens/popularDoctor.jsx:1]: sir in which file

File: src/components/backbutton.jsx
Comment (rahulagarawal7) [reply src/components/backbutton.jsx:1]: done

File: src/screens/popularDoctorScreens/popularDoctor.jsx
Comment (aayushthoughtwin) [reply src/screens/popularDoctorScreens/popularDoctor.jsx:1]: check in changes

**Pull request merged**